examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -39,132 +45,147 @@
-      oasis_dataset.gray_matter_maps[0])  # 3D data
-print('First white-matter anatomy image (3D) is located at: %s' %
-      oasis_dataset.white_matter_maps[0])  # 3D data
-
-#############################################################################
-# Preprocess data
-# ----------------
-nifti_masker = NiftiMasker(
-    standardize=False,
-    smoothing_fwhm=2,
-    memory='nilearn_cache')  # cache options
-gm_maps_masked = nifti_masker.fit_transform(gray_matter_map_filenames)
-n_samples, n_features = gm_maps_masked.shape
-print("%d samples, %d features" % (n_subjects, n_features))
-
-############################################################################
-# Prediction pipeline with ANOVA and SVR
-# ---------------------------------------
-print("ANOVA + SVR")
-# Define the prediction function to be used.
-# Here we use a Support Vector Classification, with a linear kernel
-from sklearn.svm import SVR
-svr = SVR(kernel='linear')
-
-# Dimension reduction
-from sklearn.feature_selection import VarianceThreshold, SelectKBest, \
-        f_regression
-
-# Remove features with too low between-subject variance
+# The features with too low between-subject variance are removed using
+# :class:`sklearn.feature_selection.VarianceThreshold`.
+from sklearn.feature_selection import VarianceThreshold
 variance_threshold = VarianceThreshold(threshold=.01)
+gm_maps_thresholded = variance_threshold.fit_transform(gm_maps_masked)
+gm_maps_masked = variance_threshold.inverse_transform(gm_maps_thresholded)
+
+# Then we convert the data back to the mask image in order to use it for 
+# decoding process
+mask = nifti_masker.inverse_transform(variance_threshold.get_support())
+###############################################################################
+# Training the decoder
+# ---------------------
+
+# To save time (because these are anat images with many voxels), we include
+# only the 5-percent voxels most correlated with the age variable to fit.
+# Also, we set memory_level=2 so that more of the intermediate computations
+# are cached. Also, you may pass and n_jobs=<some_high_value> to the
+# DecoderRegressor class, to take advantage of a multi-core system. We also
+# want to set mask hyperparameter to be the mask we just obtained above.
+from nilearn.decoding import DecoderRegressor
+decoder = DecoderRegressor(estimator='svr', mask=mask,
+                           screening_percentile=5,
+                           memory_level=2,
+                           memory='nilearn_cache')  # cache options
+# Fit and predict with the decoder
+decoder.fit(gm_imgs_train, age_train)
+
+# Sort test data for better visualization (trend, etc.)
+perm = np.argsort(age_test)[::-1]
+age_test = age_test[perm]
+gm_imgs_test = np.array(gm_imgs_test)[perm]
+age_pred = decoder.predict(gm_imgs_test)
+
+prediction_score = np.mean(decoder.cv_scores_['beta'])
+
+print("=== DECODER ===")
+print("r2 for the cross-validation: %f" % prediction_score)
```

Comment:
```suggestion
print("explained variance for the cross-validation: %f" % prediction_score)
```
I'd rather call this explained variance: "r2" is confusing for certain people, as they think that it is a squared correlation coefficient.